src/frontend/js/hooks/useResources/index.spec.tsx

```diff
@@ -485,7 +485,7 @@ describe('useResource (non-omniscient)', () => {
     expect(result.current.meta).toStrictEqual({
```

Comment:
Actually, I do this change because I realized the api endpoint was returning `next` and `previous` but in our codebase we were using different property name. So this commit does not impact the previous one.